### PR TITLE
bmptopnm: add page

### PR DESCRIPTION
**Pull request by debghs**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/bmptopnm.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# bmptopnm
+
+> Convert a BMP file into a PBM, PGM, or PNM image.
+> More information: <https://netpbm.sourceforge.net/doc/bmptopnm.html>.
+
+- Generate the PBM, PGM, or PNM image as output for a Microsoft Windows or OS/2 BMP file as input:
```
Comment (kbdharun) [inline pages/common/bmptopnm.md:6]:
```suggestion
- Generate the PBM, PGM, or PNM image as output, for Windows or OS/2 BMP file as input:
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by debghs** [comment]
@gutjuri could you please review this PR ??

**Review by gutjuri — COMMENTED** [review]
hello, I have one point for improvement

**Comment by debghs** [comment]
@gutjuri I have committed the suggestions, please check.

**Review by gutjuri — APPROVED** [review]

File: pages/common/bmptopnm.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# bmptopnm
+
+> Convert a BMP file into a PBM, PGM, or PNM image.
+> More information: <https://netpbm.sourceforge.net/doc/bmptopnm.html>.
+
+- Generate the PBM, PGM, or PNM image as output, for Windows or OS/2 BMP file as input:
+
+`bmptopnm {{path/to/file.bmp}}`
+
+- Report contents of the BMP header to stderr:
```
Comment (kbdharun) [inline pages/common/bmptopnm.md:10]:
```suggestion
- Report contents of the BMP header to `stderr`:
```
Just a minor suggestion and this PR is GTG.

**Comment by debghs** [comment]
@kbdharun , @gutjuri I have committed all the suggestions, kindly merge this pr.

**Pull request merged**